gapis/api/vulkan/api/descriptor.api

```diff
@@ -643,13 +643,13 @@ cmd void vkUpdateDescriptorSets(
   for _ , _ , c in copies {
     srcSet := DescriptorSets[c.SrcSet]
     dstSet := DescriptorSets[c.DstSet]
-    srcBinding := switch (c.SrcBinding in srcSet.Bindings) {
+    srcBinding := switch srcSet.Bindings[c.SrcBinding] == null {
       case false:
```

Comment:
For readability, I would prefer keeping `(c.SrcBinding in srcSet.Bindings)` and inverse the true/false of the `case` expressions.

You would also need to mirror this style by changing the condition on line 576

Reply:
`c.SrcBinding in srcSet.Bindings` is NOT semantically equivalent to `srcSet.Bindings[c.SrcBinding] == null`. 
The second being correct in more cases.

Reply:
Oh ok, so we have to keep the `== null` test.